docs/platforms/kotlin-multiplatform/troubleshooting.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve edge cases."
+sidebar_order: 9000
+---
+
+This document covers common issues you may encounter when using the Sentry Kotlin Multiplatform SDK and provides steps to troubleshoot.
+
+If you need additional help, you can [ask on GitHub](https://github.com/getsentry/sentry-kotlin-multiplatform/issues/new/choose). Customers on a paid plan may also contact support.
+
+## Tests not working
+
+If you configured the Sentry Kotlin Multiplatform SDK and tests are still failing with the following error: `ld: framework 'Sentry' not found` then follow these steps for the workaround:
+
+<Note>
+This example shows you how to fix the issue for the iOS simulator target, but the same approach can be used for other targets.
+</Note>
+
+1. Download the Sentry xcframework
+
+Ensure you have a compatible Sentry xcframework. Choose the release based on the version specified in the [version compatibility table](/platforms/kotlin-multiplatform/#cocoa-sdk-version-compatibility-table), download the `Sentry.xcframework.zip`, and unzip it.
```

Comment:
Add a tab to nest this paragraph under the step above (same for the additional details under the steps below)